Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:2-slim
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user && mkdir -p /code && chown user:user /code

```

Comment:
We could set the home dir of the user to `/code` and create it with `adduser`, this would simplify the call:
`RUN adduser --home /code --disabled-password --gecos "" user`